### PR TITLE
feat: backend - movies/[id] and /popular

### PR DESCRIPTION
**Pull request by wlechowicz**
Adds /api/movies/[id] and /api/movies/popular endpoints.

### CODE REVIEW

File: pages/api/movies/popular.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *     description: Returns a list of popular movies
+ *     responses:
+ *       200:
+ *         description: list of movies
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: array
+ *       404:
+ *         description: list not found (shound't happen)
```
Comment (marcelina-hasiak) [inline pages/api/movies/popular.js:17]: #7 + #17 typo

File: lib/services/movieDb.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  return res.json();
+};
+
+/**
+ * Get a movie by Id
+ *
+ * @export
+ * @param {string|number} id Id of the movie to get
```
Comment (marcelina-hasiak) [inline lib/services/movieDb.js:49]: "id Id" mistake?

Comment (rast-mk1) [reply lib/services/movieDb.js:49]: Nah, id is repeated there for a reason: 
obraz

Comment (wlechowicz) [reply lib/services/movieDb.js:49]: First is name of param, then comes the description.

**Pull request merged**